hystrix-contrib/hystrix-codahale-metrics-publisher/src/test/java/com/netflix/hystrix/contrib/codahalemetricspublisher/HystrixCodaHaleMetricsPublisherCommandTest.java

```diff
@@ -20,27 +23,28 @@ public void setup() {
         HystrixPlugins.getInstance().registerMetricsPublisher(new HystrixCodaHaleMetricsPublisher(metricRegistry));
     }
 
-    @After
-    public void teardown() {
-        HystrixPlugins.reset();
+    @Test
+    public void testCommandSuccess() throws InterruptedException {
+        Command command = new Command();
+        command.execute();
+
+        Thread.sleep(1000);
+
+        assertThat((Long) metricRegistry.getGauges().get("test.test.countSuccess").getValue(), is(1L));
```

Comment:
Hmm ... should this also test the other gauges ?
how about a counter from each stream ?

RollingCommandEventCounterStream
RollingCommandLatencyDistributionStream
RollingCommandUserLatencyDistributionStream
RollingCommandMaxConcurrencyStream

Which stream to timeouts happen on ?